run.sh

```diff
@@ -10,26 +10,32 @@ source image.sh
 # You should have a python environment with requirements.txt installed
 source env/bin/activate
 
+isnap=36
+ptype="COLIBRE"
+
+isnap_formatted=$( printf '%04d' $isnap )
 # First up, an example for running the scripts on one at a time
-run_directory="Runs/Run1"
-run_name="Run1"
+run_directory="/cosma7/data/dp004/dc-ploe1/SIMULATION_RUNS/2020_01_COLIBRE_ref/COLIBRE_L006N0188_ref/"
```

Comment:
Please don't commit changes with absolute paths in - these are supposed to be examples